### DISCUSSION TITLE
Cleaning up the analysis folder

### DISCUSSION THREAD

**Discussion by SamikBose** [ADicksonLab/wepy#114] [General]
Hi @salotz,

Following our discussion couple of weeks back, I went back and checked my codes and etc.

The files that we (DicksonLab) routinely use for standard analysis are contig_tree and parents. I would suggest to keep these two as it is, as it saves a lot of time. 

I have an alternate code where I build the networks and calculates the transition counts. So, those two codes you can probably move if you want to do the clean-up.

**Comment by salotz** [comment]
> I have an alternate code where I build the networks and calculates the transition counts. So, those two codes you can probably move if you want to do the clean-up.

Are they merged into this repo? If not you can make a separate repo with them and we can reference them in these forums or in the FAQ or use case guide.
